app/controllers/users_controller.rb

```diff
@@ -32,8 +33,7 @@ def show
     }.map{ |obj| UserObjectiveSerializer.new(obj)}
     @objective_rerolls = @user.reroll
     @trades = @user.trades
-    event_matches = Match.where(event_id: 1)
-    @event_ranking = EventRankingEngine.new().generate_event_user_ranking(@user)
```

Comment:
Wanted to ask why this is getting removed here since it seems unrelated to the messaging stuff. I see it's in the users show action, so I'm also not clear on why it was there originally.  General cleanup since it was unnecessary?

Reply:
Ohh I had started to clean this up and then decided that was too much work... this must've just been something I forgot to add back in.

I'd like to clean up all the matches stuff since it isn't doing anything and, if we do any events in the future, I'll want to retool a lot of that functionality